<?xml version="1.0" encoding="utf-8"?>

<launch>

  <!-- Use YOLOv4 -->
  <arg name="network_param_file"         default="$(find darknet_ros_depth)/config/c14yolov4.yaml"/>
  <arg name="image" default="/camera/left/image_raw" />


  <!-- Include main launch file -->
  <include file="$(find darknet_ros_depth)/launch/mydarknet_ros.launch">
    <arg name="network_param_file"    value="$(arg network_param_file)"/>
    <arg name="image" value="$(arg image)" />
  </include>

</launch>
